packages/server/src/routes/grants.js

```diff
@@ -103,8 +104,20 @@ router.get('/:grantId/grantDetails', requireUser, async (req, res) => {
 
 router.get('/closestGrants/:perPage/:currentPage', requireUser, async (req, res) => {
     const { perPage, currentPage } = req.params;
-    const rows = await db.getClosestGrants({ agency: req.session.selectedAgency, perPage, currentPage });
-    res.json(rows);
+    const { data, pagination } = await db.getClosestGrants({ agency: req.session.selectedAgency, perPage, currentPage });
+
```

Comment:
We display the names of interested agencies in the "Upcoming Closing Dates" table. Previously, we were retrieving these names by making a series of calls to /grants/:grantId/interested once the /grants/closestGrants call had returned. Now, we'll wrap all of that data fetching into the /grants/closestGrants call itself.